components/layout/display_list_builder.rs

```diff
@@ -1866,6 +1867,23 @@ impl ListItemFlowDisplayListBuilding for ListItemFlow {
     }
 }
 
+pub trait FlexFlowDisplayListBuilding {
+    fn build_display_list_for_flex(&mut self,
+                                   display_list: Box<DisplayList>,
+                                   layout_context: &LayoutContext);
+}
+
+impl FlexFlowDisplayListBuilding for FlexFlow {
+    fn build_display_list_for_flex(&mut self,
+                                   display_list: Box<DisplayList>,
+                                   layout_context: &LayoutContext) {
+        // Draw the rest of the block.
```

Comment:
There may not be any need for this function—just call `build_display_list_for_block` directly in `FlexFlow::build_display_list`. Unless there will actually need to be special painting behavior for flexbox in the future?